### ISSUE TITLE
Profile with EAS won't load - Can't install EAS on fresh profile

### ISSUE THREAD

**Issue by vesperax** [jobisoft/EAS-4-TbSync#155]
## Your environment

TbSync version:3.0.1
EAS-4-TbSync version:2.0.1
Thunderbird version: 91.3.2 (64-bit)

## Expected behavior
...It's my work computer, so, on friday everything was working OK - I had connected two MS mail acounts (One Office 365, one Exchange) - send/receive and sync was working normally, with TbSync and Provider for Exchange, and a few other extensions.

## Actual behavior
...Today I couldn't start Thunderbird - so, I've created a fresh profile, and installed just TbSync. Then I went on to install Provider for Exchange, but whenever I click "Add" - nothing happens, the extension just doesn't get installed. I tried downloading the xpi file, and install from folder - no luck. The only way I could test the extension was by using "Debug Add-ons" and then the temporary install somehow works - I've installed the Provider for Exchange from file. This way the extension was temporary loaded, I could even add the office 365 account, but, of course, this lasts only until I close Thunderbird, since it's temporary...  

## Steps to reproduce
...Not sure - the obvious problem for me is that I can't install the extension on a fresh profile, or start a profile where I previously had this extension installed. 
I've created a debug log, but all I can see is (bear in mind that Exchange ActiveSync is just temporary loaded, not installed):

** Mon Nov 22 2021 12:54:13 GMT+0100 (srednjoeuropsko standardno vrijeme) **
[TbSync init] : Done
** Mon Nov 22 2021 12:54:31 GMT+0100 (srednjoeuropsko standardno vrijeme) **
[Loaded provider] : eas::Exchange ActiveSync (2.0.1)

To help resolving your issue, enable debug logging (TbSync Account Manager -> Help) and send me the debug.log via e-mail (use the title of your issue as subject of the email).

**Comment by jobisoft** [comment]
Could be a localisation issue. I cannot confirm this here with the mentionhed versions and en-US. What locale have you set? Can you try a different one?

**Comment by vesperax** [comment]
Just a few typos...

> Could be a localization issue. I cannot confirm this here with the mentioned versions and en-US. What locale have you set? Can you try a different one?

I have an English version of Thunderbird, and even my Windows and Office are set to use English as display language (for menus and everything), so this should probably not be the problem, although regional settings are set to Croatian. And I didn't change a thing on my PC since friday... I'm just not sure, since the PC was on during the weekend (I just restarted it this morning, since I couldn't open Thunderbird) - maybe Thunderbird was updated... (there were no Windows updates installed). It' just - I closed Thunderbird on friday afternoon and I couldn't start it on monday morning -and however I try to install EAS on a fresh profile, it just doesn't work.
